suites/load-testing/fence/presigned-url.js

```diff
@@ -0,0 +1,32 @@
+import { check } from "k6";
+import http from "k6/http";
+
+//const TARGET_ENVIRONMENT = process.env.GEN3_COMMONS_HOSTNAME;
```

Comment:
Dunno if we should remove the commented out lines to avoid confusion?

Reply:
Good catch! I will remove them to avoid the case of confusion as we are not going to use them anymore! Keeping them can confuse.